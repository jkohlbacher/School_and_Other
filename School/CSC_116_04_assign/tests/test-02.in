it
ba
ki
za
ze
ca
li

