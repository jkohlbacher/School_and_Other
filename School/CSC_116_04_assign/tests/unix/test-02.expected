Read 178691 words.
Tiles: [it]
Solution: 1 [it] Score [2]
Solution: 2 [ti] Score [2]
Tiles: [ba]
Solution: 1 [ab] Score [4]
Solution: 2 [ba] Score [4]
Tiles: [ki]
Solution: 1 [ki] Score [6]
Tiles: [za]
Solution: 1 [za] Score [11]
Tiles: [ze]
Tiles: [ca]
Tiles: [li]
Solution: 1 [li] Score [2]
Tiles: []
